packages/manager/src/constants/API_ENDPOINTS.ts

```diff
@@ -37,6 +38,7 @@ export const API_ENDPOINTS: APIEndpoints = (() => {
 				RepositoryService: addTrailingSlash(
 					process.env.repository_api ?? "https://repository.internal.wroom.io/",
 				),
+				LocaleService: addTrailingSlash("https://locale.internal.wroom.io/"),
```

Comment:
⚠️ Here it's the dev mode, and it should take env variable, you can have a fallback as done just above for repository service, but when running in dev mode with all other env variables it would not work with wroom.io.

And so at the same time you need to ensure that locale service env variable is available in the dev env: you can ask a question in Obelix why it was never done. 
See an example how it was done for repository service: https://github.com/prismicio/prismicdev-cli/pull/59

Reply:
fixed, 
PR to add endpoint to dev env here: https://github.com/prismicio/prismicdev-cli/pull/60